core/src/transaction_status_service.rs

```diff
@@ -58,7 +61,7 @@ impl TransactionStatusService {
         let slot = bank.slot();
         for (
             (_, transaction),
-            (status, hash_age_kind),
+            (status, nonce_rollback),
```

Comment:
nice, I like these names better :)

Reply:
I kinda hate that `NonceRollback{Partial,Full}` have overloaded meaning, but thesaurus.com was failing me.  Easily fixable later!